packages/flutter/lib/src/cupertino/dialog.dart

```diff
@@ -766,33 +1338,59 @@ class _RenderCupertinoDialog extends RenderBox {
   }
 
   @override
-  bool hitTestChildren(BoxHitTestResult result, { required Offset position }) {
+  bool hitTestChildren(BoxHitTestResult result, {required Offset position}) {
```

Comment:
Here's another formatting change.  The space between `{` and `required` should be there.  I can't find this in the style guide explicitly but it's clear in many examples.